l10n_fr_ecotaxe/views/product_template_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright (C) 2020 Akretion (http://www.akretion.com/)
+    @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+-->
+<odoo>
+    <record id="product_template_form_ecotaxe" model="ir.ui.view">
+        <field name="name">product.template.ecotaxe.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//label[@for='list_price']" position="before">
+                <field name="ecotaxe_classification_id" />
+                <field
+                    name="ecotaxe_amount"
+                    widget="monetary"
```

Comment:
I'd change this field type to `Monetary` on the python file, then there's no need to specify `widget="monetary"`
(same for the other cases)